Node/src/TeamsChatConnector.ts

```diff
@@ -37,24 +37,27 @@ import * as util from 'util';
 import * as msRest from 'ms-rest';
 import RemoteQuery = require('./RemoteQuery/teams');
 import RestClient = require('./RemoteQuery/RestClient');
-import { ChannelAccount, ChannelInfo, ComposeExtensionQuery, IComposeExtensionResponse, ComposeExtensionParameter, ComposeExtensionResponse } from './models';
+import { ChannelAccount, ChannelInfo, ComposeExtensionQuery, IComposeExtensionResponse, ComposeExtensionParameter, ComposeExtensionResponse, O365ConnectorCardActionQuery } from './models';
 
 var WebResource = msRest.WebResource;
 
 export type ComposeExtensionHandlerType = (event: builder.IEvent, query: ComposeExtensionQuery, callback: (err: Error, result: IComposeExtensionResponse, statusCode: number) => void) => void;
+export type O365ConnectorCardActionHandlerType = (event: builder.IEvent, query: O365ConnectorCardActionQuery, callback: (err: Error, result: any, statusCode: number) => void) => void;
```

Comment:
Can we make "status" optional? The callback passed to dispatchEventOrQuery has "status" optional, and it's annoying to keep having to say 200.
(Same for ComposeExtensionHandlerType, for consistency).